README.md

```diff
@@ -163,7 +266,9 @@ You can also interact through [GitHub issues](https://github.com/open-source-soc
 
 Add **Open Source Society University** to your [Linkedin](https://www.linkedin.com/school/11272443/) and [Facebook](https://www.facebook.com/ossuniversity) profile!
 
+
 ## Team
 
-* **Curriculum Maintainer**: [Waciuma Wanjohi](https://github.com/waciumawanjohi)
+* **Curriculum Maintainer**: [Waciuma Wanjohi](https://github.com/waciumawanjohi3)
+* **Refined By** [SamDc73](https://github.com/SamDc73)
```

Comment:
The link to the contributors page is here so that it is easy for interested parties to see who is contributing. That contributing list is auto maintained by Github.